src/_help/publish-documentation/create-and-manage-documentation.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Create and manage documentation
+---
+
+- TOC
+{:toc}
+
+## Where can I find my documentation?
+
+By default, accessing your dashboard displays your personal account, which lists all your documentations.
+![](/images/help/personal-account.png)
+
+## Create a new documentation
+
+Creating documentation is always done from your dashboard, using the button of the same name. You will then need to provide some essential information:
+
+![](/images/help/doc-creation.png)
+
+- Name: The name of the documentation as it will appear to those who have access to it.
+- Hub: Here, you can specify whether you want to link this documentation to an existing hub or not. Hubs are collections of documentation.
+- Slug: The slug allows you to define the URL of your documentation, in the format `https://bump.sh/your-username/your-slug`.
+- Documentation Access: Documentation can be either public and accessible to everyone or private, visible only to you and invited members.
```

Comment:
Should we use a table here?
Also, can we update the "Create a new hub" page to make it consistent with this one? (in another PR, of course) We don't have this list of attributes on the other page.